api/src/utilities/seed.js

```diff
@@ -66,9 +67,9 @@ const createPublication = async (title, content) => {
    	        language: 'en',
             licence: 'CC_BY',
             content: content || title,
-            description: title || ''
+            description: title.substr(0,160) || ''
```

Comment:
this was juSt to get more publications uploading, we want to swap it to 'content' instead